idm/pkg/command/server.go

```diff
@@ -103,22 +115,38 @@ func bootstrap(logger log.Logger, cfg *config.Config, srvcfg server.Config) erro
 	}
 
 	// Prepare the initial Data from template. To be able to set the
-	// supplied admin password
-	tmpl, err := template.New("baseldif").Parse(baseldif)
+	// supplied service user passwords
+	tmpl, err := template.New("baseldif").Parse(idm.BaseLDIF)
 	if err != nil {
 		return err
 	}
 
+	for i, _ := range serviceUsers {
```

Comment:
Do not omit the second value when looping over slices. Do either
`for i := range serviceUsers` (then `i` is an `int`) or
`for _, u := range serviceUsers` (then `u` is a `svcUser`)

Imho the second approach is much more readable and therefore preferable.

Reply:
I'll need the access by index. As I am updating the list items in place. But yeah, the `_` is superfluous, I'll remove that.

Reply:
done